### PR TITLE
Adds analytics_redirect_uri field to site fields.

### PR DESCRIPTION
**Pull request by ryanwelcher**
## Summary

Addresses issue #1254

## Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: tests/phpunit/integration/Core/Authentication/Google_ProxyTest.php [hunk 1]
```diff
@@ -26,11 +26,12 @@ public function test_get_site_fields() {
...
-				'action_uri' => admin_url( 'index.php' ),
-				'name'       => get_bloginfo( 'name' ),
-				'return_uri' => $context->admin_url( 'splash' ),
-				'admin_root' => parse_url( admin_url( '/' ), PHP_URL_PATH ),
+				'url'                    => home_url(),
+				'action_uri'             => admin_url( 'index.php' ),
+				'name'                   => get_bloginfo( 'name' ),
+				'return_uri'             => $context->admin_url( 'splash' ),
+				'admin_root'             => parse_url( admin_url( '/' ), PHP_URL_PATH ),
+				'analytics_redirect_uri' => add_query_arg( 'gatoscallback', 1, home_url() ),
```
Comment (adamsilverstein) [inline tests/phpunit/integration/Core/Authentication/Google_ProxyTest.php]: This test should probably use untrailingslashit on the home url, like the function it is testing, right? will this pass if the test site home URL has a trailing slash?

Comment (ryanwelcher) [reply tests/phpunit/integration/Core/Authentication/Google_ProxyTest.php]: Thanks for catching that @adamsilverstein! I've updated the test.

**Comment by felixarntz** [comment]
@ryanwelcher This looks solid, could you rebase these commits against `refactor/analytics-data-store` though? Sorry, that probably got lost, but all GA provisioning work should be against that branch.

**Comment by ryanwelcher** [comment]
@felixarntz I pushed up replacement PR here #1315.

**Comment by felixarntz** [comment]
Closed in favor of #1315

**Pull request closed without merging**